edge_test/vault_test.go

```diff
@@ -0,0 +1,40 @@
+package edge_test
+
+import (
+	"context"
+	"flag"
+	"os"
+	"testing"
+
+	"github.com/minio/kes/edge"
+)
+
+var vaultConfigFile = flag.String("vault.config", "", "Path to a KES config file with Vault SecretsManager config")
+
+func TestVault(t *testing.T) {
+	if *vaultConfigFile == "" {
+		t.Skip("Vault tests disabled. Use -vault.config=<config file with Vault SecretManager config> to enable them")
+	}
+	file, err := os.Open(*vaultConfigFile)
+	if err != nil {
+		t.Fatal(err)
+	}
+	srvrConfig, err := edge.ReadServerConfigYAML(file)
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	store, err = srvrConfig.KeyStore.Connect(context.Background())
+	if err != nil {
+		t.Fatal(err)
+	}
+
+	os.Setenv("VAULT_SKIP_VERIFY", "true")
```

Comment:
We shouldn't set this. If we run the test with TLS we should set the `ca` field in the config file.